resources/views/partials/forms/edit/company-select.blade.php

```diff
@@ -1,23 +1,20 @@
 <!-- Company -->
+    <!-- full company support is enabled and this user is not a superadmin -->
 @if (($snipeSettings->full_multiple_companies_support=='1') && (!Auth::user()->isSuperUser()))
     <!-- full company support is enabled and this user isn't a superadmin -->
     <div class="form-group">
     {{ Form::label($fieldname, $translated_name, array('class' => 'col-md-3 control-label')) }}
         <div class="col-md-6">
-            <select class="js-data-ajax" data-endpoint="companies" data-placeholder="{{ trans('general.select_company') }}" name="{{ $fieldname }}" style="width: 100%" id="company_select" aria-label="{{ $fieldname }}">
-                @if ($company_id = old($fieldname, (isset($item)) ? $item->{$fieldname} : ''))
-                    <option value="{{ $company_id }}" selected="selected" role="option" aria-selected="true"  role="option">
```

Comment:
Don't we need a similar `value` to this, if we want to propagate the Company ID down into the asset itself?